### ISSUE TITLE
Personal take on terminology

### ISSUE THREAD

**Issue by thevangelist** [WICG/open-ui#50]
Hello earthlings. ✋ 

Interesting work you are doing here.

I hereby tell my personal opinion on the term "Design System".

I think "Live Style Guide", or "Living Style Guide" was used before to refer to what is referred as DS nowadays. I think it was a better alternative, but the best alternative is "Component library". Maybe marketing thought it's more smashing to use "Design System" - Component library sounds lame.

I think the whole industry skewed at some point during couple of years, when considering the terminology. I am unsure at which point though. The common description seems to be something similar you have on your /contribute page.

"A design system is a collection of assets and components used to create and maintain UIs on multiple platforms.".

I don't think so.  Design systems don't exist in vacuums. 

I think it also includes people, processes, communication, tools and deliverables. Like most _systems_ do. 

### What I do think is that:

* A component library is collection of assets and components used to create and maintain UIs on multiple platforms. 
* Component libraries are a part of a DS, an integral part.
* Component libraries may include UI patterns (how to use the components in correct ways, how to make pages that are following our UI pattern guidelines)
* Component libraries may or may not include branding guidelines and similar information that exist in style guides (sometimes it's hard to tell which is which)
* DS's are affected by surrounding topics, such as money paid by customer or Coronavirus.
* DS includes a design process ("process is a set of recurrent or periodic activities that interact to produce a result." -Wikipedia)
  * Making layouts
  * Reviewing components, for example.

### Some illustration here

![Untitled](https://user-images.githubusercontent.com/3482617/76385657-f10bff80-636a-11ea-89d8-be7ed0b5bff6.png)

### Examples of design systems (starting from smaller / least mature from more mature):

* One-man project where a someone draws their initial sketches to napkin and develops them to production with VIM. You could argue that there is a design system here, if he places similar buttons in similar manner, they just aren't documented
* Two-man project where the designer draws layouts with Sketch based on Sketch symbols and sends them to Dev across the globe.
  * What is included in the system here is: The communication tool and language, deliverables, etc
(... lots of examples here inbetween -  with different tools, makers, processes, communication medium, deliverables, pipelines,...)
* Some big system, like: https://designsystem.digital.gov/

### Outro

I ran out of coffee so I'm gonna get some. ☕️ 

I am sure we can't change the industry overnight - and this was just my personal take on the topic. 

**TLDR: I hope the term Design System will refer "System" instead of "Component library" in future.**

**Comment by gregwhitworth** [comment]
Thanks for the feedback. Is the primary request here that you're asking that we not refer to Open UI as a design system but that of a component library?

**Comment by LJWatson** [comment]
If I understand the aim of OpenUI properly, it is to document the generally agreed features, characteristics, properties and behaviours of components? If that's so, it feels more like a collection of blueprints than either a design system or a component library.

**Comment by gregwhitworth** [comment]
I'd note that I don't view Open UI as a design system but it is pulling in frameworks and people that work on design systems to inform the states, behaviors, anatomy, naming conventions of common components within component libraries. Any component library or platform that builds using the Open UI blueprints (to @LJWatson point) will allow for easier transition of its workers, interoperability across them (including the web platform) and also bubble up what necessary accessibility requirements (through links to ARIA/WICAG & possible spec modifications). Thanks for the feedback.

**Comment by thevangelist** [comment]
Hi, 

Commenting a bit late. Just wanted to raise awareness on the terminology.

Blueprints sounds better IMHO - Can't come up with a better name atm.

**Comment by gregwhitworth** [comment]
@thevangelist thanks, this is a term I've been using for the final output spec with folks while discussing Open UI. I don't think however we can replace the inputs into production of these blueprints.
